.vscode/extensions.json

```diff
@@ -0,0 +1,11 @@
+{
+  "recommendations": [
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
+    "executablebookproject.myst-highlight",
+    "ms-python.python",
+    "ms-python.vscode-pylance",
+    "redhat.vscode-yaml",
+    "yzhang.markdown-all-in-one"
```

Comment:
Where do these recommendations come from? Form you? from myst?

Reply:
Sorry, yes I added these recommendations so they are suggested once you open this repo in VSCode.
https://code.visualstudio.com/docs/editor/extension-marketplace#_workspace-recommended-extensions
The idea is that you standardise the dev environment where possible (in case you are using VSCode).

Could remove them again if you prefer, same goes for the `.vscode/settings.json`.